### ISSUE TITLE
The aria-labels of checkboxes in the Select component appears as [object Object] if the label is a node.

### ISSUE THREAD

**Issue by jlevesque** [grommet/grommet#1514]
The aria-labels of checkboxes in the Select component appears as [object Object] if the label is a node. As mention in the docs, the options of the select component can receive an array of strings or of objects. The label attribute of the object can be a string or a node, but if it's a node it causes some issues. If the inline and multiple attributes are true, the aria-label of the checkbox is [object Object]. 

Since the screen reader prioritized the aria-label over the <label>, it will only read the aria. However, after some investigation, if there is no aria-label at all, the accessibility requirements are still ok and the screen reader will read the text of the label tag. 

Also I've checked with the radiobutton and the issue doesn't occurs, just so you know.

If you have any idea how it can be fixed, I'll be glad to help.

### Expected Behavior
The aria-label should be a string 

### Actual Behavior
The aria-label is a rendered object

### URL, screen shot, or Codepen exhibiting the issue
<img width="651" alt="screen shot 2017-07-26 at 11 32 55 am" src="https://user-images.githubusercontent.com/6978875/28629722-44695c54-71f6-11e7-93df-29e3c19d70bb.png">

### Your Environment
* Grommet version: 1.5.0

**Comment by alansouzati** [comment]
I'm about that this is a bug!

I'm really glad to see that you are testing this stuff. Let me work on a fix for this today.

**Comment by alansouzati** [comment]
Fixed by https://github.com/grommet/grommet/commit/5109f7ea05d2733b1a52228308b159e2907ff4dc
